bad_code.java

```diff
@@ -0,0 +1,54 @@
+public class BadCodeExample {
+    
+    public static void main(String[] args) {
+        ChildClass child = new ChildClass();
+        child.doSomething();
+        child.doSomethingElse();
+    }
+}
+
+class ParentClass {
+    public int someNumber = 10;
```

Comment:
Consider using a constructor in `ParentClass` and `ChildClass` to initialize class fields instead of direct field initialization. This approach is more flexible and maintainable. [medium]